index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="css/style.css"/>
+    <title>Ezra Yehuwalashet</title>
+  </head>
+<body>
+  <!-- Main page section -->
+  <section id="main-page">
+    <header>
+      <nav id="navbar">
+        <a href="#" class="my-logo">Ezra</a>
+        <button class="hamburger">
+          <i class="menuIcon material-icons">menu</i>
+        </button>
+      </nav>
+    </header>
+    <main class="headline">
+      <h1 class="title">I'm Ezra <br> Glad to see you!</h1>
+      <p class="content">
+        I am a software develper. I can help you build a product, feature or
+        website. Look through some of my work and expereince. If you like what
+        you see and have a project you need coder, don't husitate to contact
+        me.
+      </p>
+      <h3 class="cover-links">LET'S CONNECT</h3>
+      <ul class="cover-socials">
+        <li><a href="https://github.com/ezraermy" class="fa fa-github"></a></li>
+        <li><a href="https://www.linkedin.com/in/ezra-yehuwalashet/" class="fa fa-linkedin"></a></li>
+        <li><a href="#" class="fa fa-google"></a></li>
+        <li><a href="https://twitter.com/ezraermy" class="fa fa-twitter"></a></li>
+        <li><a href="https://facebook.com/ezraermias" class="fa fa-facebook"></a></li>
+      </ul>
+    </main>
+  </section>
+
+   <!-- First Project -->
+   <section id="card-page" class="work-page-right">
```

Comment:
Please use`<section>` for the container of your works cards as requested on the project requirements. And please use display:grid for the container and use display:flex for each individual card.
<img width="1004" alt="Screenshot 2023-01-13 at 12 07 07 AM" src="https://user-images.githubusercontent.com/25887253/212181756-c2659697-f173-4ab5-a519-803373dbe6ce.png">